LookupAnything/Framework/Lookups/Items/ItemSubject.cs

```diff
@@ -467,10 +467,14 @@ private string GetTypeValue(Item item)
 
             try
             {
-                Crop crop = new(seed.ItemId, 0, 0, location ?? Game1.getFarm());
```

Comment:
Similarly I suggest changing the condition above to avoid item ID collisions:
```cs
// current
if (seed is not SObject obj || obj.bigCraftable.Value)

// suggested
if (seed.HasTypeId(ItemRegistry.type_object))
```

Reply:
Done (inverted HasTypeId to preserve the short circuit behavior)